arviz/tests/base_tests/test_data.py

```diff
@@ -1262,6 +1262,37 @@ def test_empty_inference_data_object(self):
         os.remove(filepath)
         assert not os.path.exists(filepath)
 
+    def test_dask_chunk_group_kwds(self):
```

Comment:
These are great, but I would move them to their own file, i.e. `test_data_dask` or something like that in order to follow the same pattern as other optional dependencies. See for example https://github.com/arviz-devs/arviz/blob/main/arviz/tests/base_tests/test_stats_numba.py#L17. This allows contributors to run the test suite without needing the optional dependencies nor having to go over each failed test to make sure only the ones with optional dependencies are failing.

Reply:
makes sense.  Just added `test_data_dask.py`